docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: '3'
+
+services:
+  springboot-docker-compose-app-container:
+    image: neutralizer/book-manager-dockerize-maven:1.0
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    depends_on:
+      - mysqldb
+    ports:
+      - "8080:8080"
+
+  prometheus:
+    image: prom/prometheus:v2.6.1
+    ports:
+      - "9090:9090"
+
+  grafana:
+    image: grafana/grafana:5.4.3
+    depends_on:
+      - prometheus
+    ports:
+      - "3000:3000"
+
+  mysqldb:
+    image: mysql:8
+    environment:
+      - MYSQL_ROOT_PASSWORD=root
```

Comment:
it's ok for now, but we should use some secret storage for sensitive data

Reply:
Added in todo list